### ISSUE TITLE
Error on Windows when the taskbar is on left or right position

### ISSUE THREAD

**Issue by pranit123** [maxogden/menubar#233]
[//]: # "Hey there and thank you for using Menubar's issue tracker!"
[//]: # "Please fill in these information to describe your issue."

### Description
For top and bottom tray positions, the window position is calculated correctly. But when the taskbar is on left or right, a runtime error occurs in the `showWindow` function.

### Steps to Reproduce the Problem

1. Create a menubar without passing any `windowPosition`
2. Place the taskbar to the left or right on Windows 10
3. Run the app and click on the tray icon

### Expected Behaviour
The window should be created near the taskbar icon when it is clicked

### Actual Behaviour
The app crashes

### Specifications

- Menubar version: 6.0.5
- Platform: Windows 10
- Electron version: 4.1.3

**Comment by amaurymartiny** [comment]
Thanks for the error report. The error you have is it:

```
TypeError: Error processing argument at index 1, conversion failure from 426.5
```

?

**Comment by amaurymartiny** [comment]
Fixed in [menubar@6.0.6](https://www.npmjs.com/package/menubar/v/6.0.6) 🚀
